Web/starter_project_vanguard/pages/success-story/index.tsx

```diff
@@ -1,10 +1,50 @@
 import Partners from '@/components/success-story/Partners'
 import SuccessDescription from '@/components/success-story/SuccessDescription'
 import SuccessImageCard from '@/components/success-story/SuccessImageCard'
-import { SuccessStory } from '@/types/success-story'
-import successStoryjsonData from '../../data/success-story.json'
+import { useFetchSuccessStoryQuery } from '@/features/success-story/success-story-api-slice'
 export default function SucessStory() {
-  const successStories: SuccessStory[] = successStoryjsonData
+  const { data = [], isFetching } = useFetchSuccessStoryQuery()
+  if (isFetching) {
+    return (
+      <div className=" h-screen flex justify-center items-center">
+        <div
+          className="inline-block h-8 w-8 animate-spin rounded-full border-4 border-solid border-current border-r-transparent align-[-0.125em] motion-reduce:animate-[spin_1.5s_linear_infinite]"
+          role="status"
+        >
+          <span className="!absolute !-m-px !h-px !w-px !overflow-hidden !whitespace-nowrap !border-0 !p-0 ![clip:rect(0,0,0,0)]">
+            Loading...
+          </span>
+        </div>
+      </div>
+    )
+  }
+  if (data.length === 0) {
+    return (
+      <section className="flex items-center h-full p-16 dark:bg-gray-900 dark:text-gray-100">
+        <div className="container flex flex-col items-center justify-center px-5 mx-auto my-8">
+          <div className="max-w-md text-center">
+            <h2 className="mb-8 font-extrabold text-9xl dark:text-gray-600">
+              <span className="sr-only">Error</span>404
+            </h2>
+            <p className="text-2xl font-semibold md:text-3xl">
+              Sorry, we couldn't find this page.
+            </p>
+            <p className="mt-4 mb-8 dark:text-gray-400">
+              But dont worry, you can find plenty of other things on our
+              homepage.
+            </p>
+            <a
```

Comment:
In React, it is often recommended to use the `Link` component instead of the traditional `a` tag when working with links or navigation.